python-django webapp

zlyxero opened the pull request.

zlyxero committed:
Add files via upload

agrean commented:
phonenumber_field module does not exist

On Wed, Sep 12, 2018 at 6:06 PM Jeff <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>
> https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3
> Commit Summary
>
>    - Add files via upload
>
> File Changes
>
>    - *A* airtime_pap/Pipfile
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-0>
>    (14)
>    - *A* airtime_pap/Pipfile.lock
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-1>
>    (107)
>    - *A* airtime_pap/airtime_app/admin.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-2>
>    (4)
>    - *A* airtime_pap/airtime_app/apps.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-3>
>    (5)
>    - *A* airtime_pap/airtime_app/forms.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-4>
>    (20)
>    - *A* airtime_pap/airtime_app/migrations/0001_initial.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-5>
>    (27)
>    - *A* airtime_pap/airtime_app/models.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-6>
>    (27)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/contacts.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-7>
>    (8)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/create_contact.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-8>
>    (9)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/form_snippet.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-9>
>    (15)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/home.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-10>
>    (5)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/selectToSend.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-11>
>    (15)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/send_airtime.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-12>
>    (20)
>    - *A* airtime_pap/airtime_app/templates/airtime_app/signup.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-13>
>    (14)
>    - *A* airtime_pap/airtime_app/templates/registration/login.html
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-14>
>    (9)
>    - *A* airtime_pap/airtime_app/tests.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-15>
>    (3)
>    - *A* airtime_pap/airtime_app/urls.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-16>
>    (26)
>    - *A* airtime_pap/airtime_app/views.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-17>
>    (123)
>    - *A* airtime_pap/airtime_project/settings.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-18>
>    (129)
>    - *A* airtime_pap/airtime_project/urls.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-19>
>    (22)
>    - *A* airtime_pap/airtime_project/wsgi.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-20>
>    (16)
>    - *A* airtime_pap/contacts_selected
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-21>
>    (0)
>    - *A* airtime_pap/db.sqlite3
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-22>
>    (0)
>    - *A* airtime_pap/manage.py
>    <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3/files#diff-23>
>    (15)
>
> Patch Links:
>
>    -
>    https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3.patch
>    -
>    https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASvHncenYlSKq3bmq5cEGxhL5TsrokxZks5uaSLagaJpZM4Wllm8>
> .
>


-- 

*Warm Regards,*
*Kamau Abraham Kirumba*.

zlyxero committed:
AT sms API intergration and webapp design

zlyxero committed:
AT airtime API intergration and UI design updates

zlyxero commented:
Hi Abraham, 
Thanks for looking at the code.  The phonenumber_field module is an External library and is included as part of the dependencies for the app in the Pipfile. Just to be sure, I posted an update on the repo. 
The App is now deployed, kindly find it at; [](https://airtimeapp.herokuapp.com/)

Much appreciation for the feedback Abraham.

**kind regards**,
**Jeff**

zlyxero commented:
@agrean  Sorry, I omitted the app link;

 [deployed app ](https://airtimeapp.herokuapp.com)

agrean commented:
Hi Jeff
-How do I add that library. I would like to see check the back end out.
-I liked the web app deployed.

On Fri, Sep 14, 2018 at 6:30 PM Jeff <notifications@github.com> wrote:

> @agrean <https://github.com/agrean> Sorry, I omitted the app link;
>
> deployed app <https://airtimeapp.herokuapp.com>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3#issuecomment-421395764>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASvHnfhlPkq9KtDN2OqW1gyoSSmAx_BPks5ua8ulgaJpZM4Wllm8>
> .
>


-- 

*Warm Regards,*
*Kamau Abraham Kirumba*.

zikner committed:
Send airtime page updates

*Check if a user selected a contact before sending Airtime
*Re-designed the checkbox for a better user experience

zikner committed:
Updated README.md

zikner committed:
Updated README.md

zikner committed:
Updated README.md

*added a link to the deployed app

zlyxero commented:
@agrean thanks for checking out the deployed app.

I updated the README file with the app's [installation instructions](https://github.com/zlyxero/UIUXCodeChallengeSeptember2018/tree/254711276275) .. In the readme file, checkout the **solution** section.

If you follow the instructions, you will notice a 'Pipfile'.  
Inside this file, all the app dependencies will be listed, including the 'django-phonenumber_field' library. The installation instructions will guide you on how to install the library together with the other dependencies such as the 'africastalking' library

I hope that will help answer your question. If you have any more questions let me know :))

zikner committed:
Updated README.md

agrean commented:
@zlyxero 
Thanks

zlyxero commented:
You're welcome.
On Sep 18, 2018 16:41, "Abraham Kirumba" <notifications@github.com> wrote:

> @zlyxero <https://github.com/zlyxero>
> Thanks
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/AfricasTalkingTalent/UIUXCodeChallengeSeptember2018/pull/3#issuecomment-422397735>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP0cXmlEXvMieg5O7auCHUwPNpmtaMIoks5ucPgJgaJpZM4Wllm8>
> .
>
